pages/app/student-profile/[studentprofile].js

```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect } from 'react';
```

Comment:
Nit: same as teacherProfile, useEffect is imported but not used yet. Not relevant right now but be aware that we will be trying as much as possible to use SSG and SSR so useEffect will be used sparingly for files under /pages directory